demo/image-carousel-demo-app.html

```diff
@@ -0,0 +1,31 @@
+<link rel="import" href="../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../image-carousel.html">
+<link rel="import" href="../bower_components/polymer/lib/elements/dom-repeat.html">
```

Comment:
Replace `bower_component` with `..` as `polymer serve` should be able to find the correct path.

Reply:
done.